cloudify/tests/test_builtin_workflows.py

```diff
@@ -280,25 +280,22 @@ def test_filter_uninstall(self, cfy_local):
         assert states == {'deleted': 4}
 
     steps_blueprint = """
-        tosca_definitions_version: cloudify_dsl_1_3
-        imports: [minimal_types.yaml]
-        node_types:
-            type1:
-                derived_from: cloudify.nodes.Root
-        node_templates:
-            step1:
-                type: cloudify.nodes.Root
-            step2:
-                type: cloudify.nodes.Root
-                relationships:
-                    - target: step1
-                      type: cloudify.relationships.depends_on
-            step3:
-                type: cloudify.nodes.Root
-                relationships:
-                    - target: step2
-                      type: cloudify.relationships.depends_on
-    """
+tosca_definitions_version: cloudify_dsl_1_3
+imports: [minimal_types.yaml]
+node_templates:
+  step1:
+    type: cloudify.nodes.Root
+  step2:
+    type: cloudify.nodes.Root
+    relationships:
+      - target: step1
+        type: cloudify.relationships.depends_on
+  step3:
+    type: cloudify.nodes.Root
+    relationships:
+      - target: step2
+        type: cloudify.relationships.depends_on
+"""
```

Comment:
I didn't really _need_ to change the syntax on this test, but just for consistency with master, and with the new test.